enhancements/sig-architecture/82-addon-template/README.md

```diff
@@ -36,8 +36,8 @@ straightforward addon development way.
 
 - The addon agent image is still required.
 - The hosted mode addon is not supported.
-- The pre-delete-hook feature is not supported.
 - For complicated addon which has special RBAC requirement etc would still need to use addon-framework.
+- For the agent runtime, only support Deployment, not support Daemonset, Statefulset, etc.
```

Comment:
I think you support that, but cannot return status? we should clarify this. A user can set sts/deploy in the same template, but addon status only reflect status of deploy right now.

Reply:
but we only inject volumes that can access the hub cluster into the deployment.

Reply:
patl